pandas/io/excel/_util.py

```diff
@@ -23,32 +23,46 @@ def register_writer(klass):
     _writers[engine_name] = klass
 
 
-def get_default_writer(ext):
+def get_default_engine(ext, mode="read"):
     """
-    Return the default writer for the given extension.
+    Return the default reader/writer for the given extension.
 
     Parameters
     ----------
     ext : str
         The excel file extension for which to get the default engine.
+    mode : str
+        Whether to get the default engine for reading or writing.
+        Either 'read' or 'write'
 
     Returns
     -------
     str
         The default engine for the extension.
     """
+    _default_readers = {
```

Comment:
hmm maybe this could integrate with the `_writers` and `_readers` module level variables?

Reply:
@jreback I'm not sure how these are related. The _writers dict at the top of this file is simply a mapping between engine names and there class names and I don't know where to find the _readers dict. Can you please elaborate on this more?

Reply:
no i mean the _default_readers/writers could simply be module level as well. but i guess ok here too.